tests/.infra/crw-ci/nightly/k8s/single-host/Jenkinsfile

```diff
@@ -8,11 +8,11 @@ pipeline {
                 artifactNumToKeepStr: '', daysToKeepStr: '60', numToKeepStr: '30'))
     }
 
-    triggers { cron("H 2 * * *") }
+    triggers { cron("H 1 * * *") }
 
     parameters {
         string(name: 'emails',
-                defaultValue: 'dnochevn@redhat.com,kfoniok@redhat.com',
+                defaultValue: 'oshmarai@redhat.com',
```

Comment:
Could you, please, notify "dnochevn@redhat.com" as well?

Reply:
Done.